sql-cli/pyproject.toml

```diff
@@ -1,31 +1,14 @@
-[build-system]
-requires = ["flit_core ~=3.2"]
-build-backend = "flit_core.buildapi"
-
-[project]
+[tool.poetry]
 name = "astro-sql-cli"
-dynamic = ["version"]
-description = """
-Astro SDK allows rapid and clean development of {Extract, Load, Transform} workflows using Python and SQL, powered by Apache Airflow.
-"""
-
+version = "0.0.0"
+description = "Empower analysts to build workflows to transform data using SQL"
 authors = [
-    {name = "Astronomer", email = "humans@astronomer.io"},
+    "Astronomer <humans@astronomer.io>",
 ]
 readme = "README.md"
-license = {file = "LICENSE"}
-
-requires-python = ">=3.7"
-dependencies = [
-    "apache-airflow>=2.0",
-    "pandas>=1.3.4,<2.0.0",  # Pinning it to <2.0.0 to avoid breaking changes
-    "pyarrow",
-    "python-frontmatter",
-    "smart-open",
-    "SQLAlchemy>=1.3.18"
-]
-
-keywords = ["airflow", "provider", "astronomer", "sql", "decorator", "task flow", "elt", "etl", "dag"]
+license = "Apache-2.0"
+keywords = ["sql", "transform", "analyst", "workflow", "airflow", "astronomer"]
+packages = [{include = "sql_cli"}]
 classifiers = [
     "Development Status :: 5 - Production/Stable",
```

Comment:
Not your change, but curious what does this classifier mean? and if it is relevant/suitable for us?

Reply:
Yeah that should be changed to Alpha. It is used for PyPI classifiers: https://pypi.org/classifiers/